go.mod

```diff
@@ -96,3 +95,5 @@ require (
 	k8s.io/klog v0.1.0 // indirect
 	sigs.k8s.io/yaml v1.1.0 // indirect
 )
+
+go 1.13
```

Comment:
Is this something new in Go 1.13? I'm not running that yet, I believe it's still a preview release.

Reply:
Ah, it might be, the 1.13 is because I compile from tip but I'm not familiar enough with Go modules to know why the go.mod file needs a Go version. I just deleted it again.